README.md

```diff
@@ -9,68 +9,301 @@
 
-![Detect duplicate code](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Duplicate_Code_Detection_VS_Code.gif)
+### Continuous code quality feedback
+
+![Code Quality Metrics in Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_Metrics_VS_Code.gif)
+
+Sourcery gives each of your functions a quality score ranging from 0%(bad) - 100% (good) and also gives you sub-scores on Method Length, Complexity, and Working Memory so that you can figure out how to structure your code as cleanly as possible.
+
+**Method Length** is a metric is a measure of how long each method is on average. It is based on the number of nodes in the method's Abstract Syntax Tree.
+
+**Complexity** is a measure of how difficult your code is to read and understand. It is based on these principles:
+
+- Each break in the linear flow of the code makes it harder to understand
+- When these flow-breaking structures are nested they are even harder to understand
+
+**Working Memory** is a measure of the number of variables that need to be kept in your working memory as you read through the code.
+
+Sourcery will warn you if your overall quality score for a function falls below 25%.
+
+### Multi-File Analysis
+
+![Multi-File Analysis & Refactoring in Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_Example_Full_Project_Scan_VS_Code.gif)
+
+You can use Sourcery to refactor a single file, a folder, or your whole project at once. 
+
+Right-click on any item in the Explorer window and select "Sourcery → Scan for Refactorings". Sourcery will show you a notification when the scan is done & will show all of the refactorings it found in the Problems window. 
+
+Multi-File Analysis requires a Sourcery Pro subscription. To get a month free access to Sourcery Pro, **[sign up for an account on the Sourcery site](https://sourcery.ai/signup/?utm_source=VS-Code)**.
+
+### Duplicate Code Detection
+
+![Duplicate Code Detection in Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_Example_Duplicates_VS_Code.gif)
+
+Sourcery can help you find duplicate code and near-duplicate code across your whole project. 
+
+Right-click on any item in the Explorer window and select "Sourcery → Detect Clones". Sourcery will show you a notification when it's done scanning for clones & will show you all of the instances of duplication it found in the Problems window. 
+
+By default, Sourcery will flag items where at least 3 lines are duplicates or near-duplicates that occur at least twice in the scanned files. 
+
+Duplicate Code Detection requires a Sourcery Pro subscription. To get a month free access to Sourcery Pro, **[sign up for an account on the Sourcery site](https://sourcery.ai/signup/?utm_source=VS-Code)**.
+
+### GitHub Pull Request Review
```

Comment:
Does this and Sourcery CLI belong in a separate top level heading?

Reply:
I was treating them as separate "Features" of Sourcery in this section, but they could be their own separate top level heading. Not sure the best way to split out